bl.js

```diff
@@ -132,8 +142,10 @@ BufferList.prototype.copy = function copy (dst, dstStart, srcStart, srcEnd) {
 
   // copy/slice everything
   if (srcStart === 0 && srcEnd == this.length) {
-    if (!copy) // slice, just return a full concat
-      return Buffer.concat(this._bufs)
+    if (!copy) // slice, but full concat if multiple buffers
```

Comment:
not critical but braces around a multi-line would be nice, even if it's just a single statement broken over multiple lines.